src/compaction_execute_bench.rs

```diff
@@ -296,7 +297,7 @@ fn load_options() -> Options {
         val_bytes: val_bytes.parse::<usize>().expect("invalid val bytes"),
         compression_codec,
     };
-    println!("Options: {:#?}", options);
+    info!("Options: {:#?}", options);
```

Comment:
Should we continue using formatted strings or should we use structured events?
This is first time I am looking tracing crate, trying to learn what would be better.

Can you share examples of how these traces look like in the console subscriber?

Reply:
```
#[derive(Debug)]
struct Baz {
    string: String,
    int: i32,
    bool_value: bool,
    now: Instant
}
```

```
info!("Baz value {:?}", b);
```


```
2024-08-18T04:25:18.241655Z  INFO tracing_demo: Baz value Baz {
    string: "slate",
    int: 123,
    bool_value: false,
    now: Instant {
        tv_sec: 361652,
        tv_nsec: 925992000,
    },
}
```

Reply:
Not critical to this conversation, but multi-line log lines are a bummer. Is it possible to configure `log` to emit single line?

Reply:
I have remove pretty print formatting, this should emit log in single line